shared/libraries/config_protocol/include/config_protocol/config_protocol.h

```diff
@@ -45,7 +46,8 @@ enum PacketType: uint8_t
     UpgradeProtocol = 0x81,
     Rpc = 0x82,
     ServerNotification = 0x83,
-    InvalidRequest = 0x84
+    InvalidRequest = 0x84,
+    NoReplyRpc = 0x85
```

Comment:
Do we need to update the protocol version here? What happens when we try executing this call on an older server?

Reply:
I've updated the version number, ensuring that this new packet type won't reach older server versions. However, the server would technically just ignore these packets anyway.